tools/ci_build/amd_hipify.py

```diff
@@ -35,6 +35,9 @@ def hipify(hipify_perl_path, src_file_path, dst_file_path):
     s = s.replace("HIPBLAS_OP_T", "rocblas_operation_transpose")
     s = s.replace("HIPBLAS_OP_N", "rocblas_operation_none")
 
+    # in rocm 6.0, hipify-perl, the -roc option also maps __half -> rocblas_half which we don't want
+    s = s.replace("rocblas_half", "__half")
```

Comment:
Is it possible to hipify without the `--roc` option those sources which are targeting HIP, not ROC, instead?

Reply:
I opted to use the existing python script to undo the change because it turned out the `--roc` option has been used for a very long time and I was worried about side effects if I removed it rather than just [further] patching the output from hipify-perl.